packages/ember-routing-htmlbars/lib/keywords/closure-action.js

```diff
@@ -67,6 +67,22 @@ export default function closureAction(morph, env, scope, params, hash, template,
   return s;
 }
 
+
+function extractRawAction(params, scope) {
+  var [value] = params;
+  if (value) {
```

Comment:
I'm not clear how this is doing the right thing- A stream must be `read` to get the value. @rwjblue is pretty sure the tests for this just didn't run, b/c we only run tests for PRs against `master`, `beta`, or `release`.